pyproject.toml

```diff
@@ -297,11 +297,6 @@ ignore = [
     "B904",
     "D105", # Missing docstring in magic method
     "D107", # Missing docstring in `__init__`
```

Comment:
D107 is also included in extend-select. Do you intend to remove it from the IGNORE list?
```suggestion
```

Reply:
It should be removed from extend-select then. If we include it we get about 23 errors that need to be fixed. And I think we put the docs in the class docstring instead of in __init__ by convention

Reply:
Also, I never really finished implementing this PR, so the lists are not fully complete